### ISSUE TITLE
HUE Api v2

### ISSUE THREAD

**Issue by Chris53897** [neoteknic/Phue#11]
I am not sure how much effort is needed to migrate to v2.
[Migration Guide to the new Hue API](https://developers.meethue.com/develop/hue-api-v2/migration-guide-to-the-new-hue-api/)

Not all bridges are supported.
`Just make sure to check that the bridge version is at least 1948086000.`

If somebody is interested into a migration, that would be great.

**Comment by neoteknic** [comment]
Yes, and DTLS support entertainment would be awesome (swoole needed maybe), but I dont have the time to do it now :(
